tests/test_config_flow.py

```
@@ -0,0 +1,118 @@
+            "async_entry_for_domain_unique_id",
+            return_value=None,
+        ),
+    ):
+        result = await flow.async_step_user(user_input)
+    assert result["type"] == "create_entry"
+    assert "ResMed-CPAP" in result["title"]
+    assert result["data"][CONF_USER_NAME] == "user"
+
+
+@pytest.mark.asyncio
+async def test_async_step_user_auth_error(flow):
```

Ruff (0.12.2)

Missing docstring in public function

(D103)